resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -569,10 +569,10 @@ cloudwatch.logs.open=View Log Streams
 cloudwatch.logs.no_events=No Events in Log Stream
 cloudwatch.logs.no_events_query=No Events matching the query found in Log Stream {0}
 cloudwatch.logs.log_stream_title=Stream: {0}
-cloudwatch.logs.filtered_log_stream_title=Filtered Stream: {0} from {1} to {2}
+cloudwatch.logs.filtered_log_stream_title=Filtered Stream: {0} from {1} to {2} around message "{3}"
```

Comment:
Wonder if message is going to be too long here... Whats it look like?

Reply:
Oh yeah this message is way too long, oh well at least we can have the time there